corehq/apps/app_execution/data_model.py

```diff
@@ -60,6 +60,13 @@ class CommandStep(Step):
     selected_values: list[str] = None
     """Selected values for multi-select commands"""
 
+    def to_json(self):
+        data = super().to_json()
+        for key in ["value", "id", "selected_values"]:
```

Comment:
I am reading this right? If any of those three keys is set to None with will remove it from the returned Json?

Reply:
That's correct, it's mainly to keep what's shown to the user simple.